src/main/java/team18/team18_be/apply/controller/ApplyController.java

```diff
@@ -42,6 +44,14 @@ public ResponseEntity<Void> createApplicationForm(
     return ResponseEntity.created(location).build();
   }
 
+  @Operation(summary = "특정 지원서 조회")
+  @GetMapping("/form/{applyId}")
```

Comment:
```suggestion
  @GetMapping("/{applyId}/form")
```
코드만 봤을 땐 이것이 더 어울려 보입니다.
자세한 도메인은 멘티님들이 더 잘 알고계시니 참고만 해주세요.